src/clients/kdump.rb

```diff
@@ -734,6 +734,8 @@ def cmdKdumpShow(options)
     end
 
 
+    # Only numbers are allowed as allow_mem_high and allow_mem_low values
+    ALLOC_MEM_REGEXP = /\D/
```

Comment:
I just tested:
gilson@linux-8x2s:~/yast-kdump> sudo yast2 kdump startup enable alloc_mem=R2D2
Invalid allocation memory parameter
Use 'yast2 kdump help' for a complete list of available commands.